src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -121,4 +122,9 @@ public function isAncestor(ItemInterface $item, ?int $depth = null): bool
 
         return $this->runtimeExtension->isAncestor($item, $depth);
     }
+
+    public function spaceless(string $content): string
```

Comment:
I suggest making this a static method instead, which will allow the compiled template to call it directly.

And I would definitely tag the implementation method as `@internal`